scripts/test/test-utils.sh

```diff
@@ -154,5 +156,18 @@ get_pytest_files() {
   wget -N ${MENDER_ACCEPTANCE_URL}/helpers.py -P $WORKSPACE/mender-image-tests
   wget -N ${MENDER_ACCEPTANCE_URL}/conftest.py -P $WORKSPACE/mender-image-tests
   wget -N ${MENDER_ACCEPTANCE_URL}/fixtures.py -P $WORKSPACE/mender-image-tests
+  mkdir -p $WORKSPACE/mender-image-tests/files
+  wget -N ${MENDER_ACCEPTANCE_URL}/files/test-private-EC.pem -P $WORKSPACE/mender-image-tests/files
+  wget -N ${MENDER_ACCEPTANCE_URL}/files/test-private-RSA.pem -P $WORKSPACE/mender-image-tests/files
+  wget -N ${MENDER_ACCEPTANCE_URL}/files/test-public-EC.pem -P $WORKSPACE/mender-image-tests/files
+  wget -N ${MENDER_ACCEPTANCE_URL}/files/test-public-RSA.pem -P $WORKSPACE/mender-image-tests/files
 }
 
+prepare_ssh_keys() {
+  if [ "$(stat -c %U tests/ssh-public-key-overlay/root)" != "root" ]; then
+    sudo chown -R root:root tests/ssh-public-key-overlay/root
+  fi
+  if [ "$(stat -c %a tests/ssh-priv-key/key)" != "600" ]; then
+    chmod 600 tests/ssh-priv-key/key
+  fi
```

Comment:
suggestion: nice ifs, but I would just drop them and run chown and chmod. 
doesn't chmod need sudo as well?

Reply:
When running locally, it was very annoying to get asked for sudo password often, that is why I added the ifs. I think I will leave it.

That specific chmod is done on the private key, which is owned by whoever cloned the repo, therefore does not require sudo.

NOTE: This was actually the problem for the tests to fail (incorrect permissions), the last force-push added more `chmod`s here that made the tests finally pass.